assets/javascripts/discourse/components/post-voting-comment-editor.js

```diff
@@ -0,0 +1,48 @@
+import { action } from "@ember/object";
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { inject as service } from "@ember/service";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+import { ajax } from "discourse/lib/ajax";
+
+export default class PostVotingCommentEditor extends Component {
+  @service siteSettings;
+
+  @tracked value = this.args.raw;
+  @tracked submitDisabled = true;
+
+  @action
+  updateValue(value) {
+    this.value = value;
+    this.submitDisabled =
```

Comment:
The same, what if `this.siteSettings.min_post_length` > `this.siteSettings.qa_comment_max_raw_length`?

Reply:
I don't think it's worth it, `qa_comment_max_raw_length` is a hidden setting with a default of 600 (not sure why it's hidden though) and the error messages are a good hint on how to "solve" it if it happens